src/controllers/download.attachment.landing.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { NextFunction, Request, Response } from "express";
+import { Templates } from "../model/template.paths";
+import logger from "../utils/logger";
+
+/**
+ * GET controller for download attachment landing screen
+ * @param req
+ * @param res
+ * @param next
+ */
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  const url: string = req.originalUrl.replace("download/", "");
```

Comment:
If i remember , i think this is stripping off the download prefix, in which case you could reference the value from pageUrls PREFIX + SEPARATOR?

Reply:
Replaced with the DOWNLOAD_PREFIX from pageUrls, .replace only removes the first instance of /download so it won't trim off the end as well.